change/@fluentui-react-tooltip-0c7f8278-85ad-44f1-82b3-8048cf63b2e7.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "patch: updated styles for tooltip",
```

Comment:
Please update this change comment with a better description (and begin with `fix:` instead of patch).

Reply:
What about the type, does that needs to be changed as well?